examples/resources/rafay_alertconfig/providers.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    rafay = {
+      version = "1.1.3"
+      source = "registry.terraform.io/Rafay/rafay"
```

Comment:
Change the source from registry.terraform.io/Rafay/rafay to registry.terraform.io/RafaySystems/rafay in all example files. Also, change the version.

Reply:
@sachin-rafay Updated it. Thanks for pointing out!